services/githubService.js

```diff
@@ -1,4 +1,4 @@
-const { fetch } = require("../utils/fetch");
+const axios = require("../utils/fetch");
```

Comment:
Why this change?

Reply:
I was not able to stub it using sinon js for returning mock data. So had to convert it.